src/Model/SnapUser.hs

```diff
@@ -0,0 +1,28 @@
+module Model.SnapUser where
+  
+import Data.Time.Calendar
+import Snap.Snaplet.PostgresqlSimple
+
+data SnapUser = SnapUser
+  { id :: Int
+  , name :: String
+  , email :: String
+  , birthDate :: Day
+  , password :: String
+  , genre :: String
+  , income :: Double
+  }
```

Comment:
It's a matter of taste... but you will find that most of haskell code is tabularized... like
```hs
data SnapUser = SnapUser
  { id   :: Int
  , name :: String
...
```

Reply:
@CristhianMotoche what is the convention of field naming here... for example, in persistent, if you have an entity named user, then the `name` field, will be `userName`... maybe we can do the same here WDYT?

Reply:
@dbalseiro @kinorius123 In some projects, I've seen something like this:
```haskell
data User = 
   User {
      uId :: Int
    , uName :: Text
    , uBirthday ::  Day
    }
```
Using only every uppercase letters of every word in the record as a prefix. In that way, we avoid problems with collision names.

Reply:
@kinorius123 Could you please follow that convention?